CWE/inventions/New inventions.txt

```diff
@@ -300,4 +300,40 @@ full_body_replacement = {
             health = 0.05
         }
 	}
+}
+
+land_ethic = {
+	limit = { intensive_farming = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = 2
+			year = 1980
+		}
+	}
+
+	effect = {
+		shared_prestige = 5
+		farm_rgo_eff = 0.01 #More efficient/sustainable use of land
+	}
+}
+
+smart_grid = {
+	limit = { sustainable_energy = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = 2
+			handheld_devices = 1
+		}
+		modifier = {
+			factor = 2
+			modern_backbone = 1
+		}
+	}
+	effect = {
+		factory_goods_throughput = {
+			heating_electricity = 0.05 #I think this reduces the amount of input to make an output?
```

Comment:
throughput increases rate of production so input and output are just scaled upwards proportionally.

Reply:
Alright, I'll just have it increase output for heating_electricity then. I'm not sure how else to simulate efficiency.

Reply:
Okay